src/test/java/org/broadinstitute/hellbender/tools/spark/sv/InsertSizeMetricsCollectorSparkUnitTest.java

```diff
@@ -23,7 +25,13 @@ public void testRanges1() throws Exception {
         map.put(44, 2L);
         map.put(45, 1L);
 
-        long[] calculatedBinWidths = InsertSizeMetricsCollectorSpark.computeRanges(map, 41, 13);
+        Histogram<Integer> hist = new Histogram<>("dummy", "test");
+        for(final int size: map.keySet()){
+            hist.prefillBins(size);
+            hist.increment(size, map.get(size));
+        }
+
+        long[] calculatedBinWidths = InsertSizeMetricsCollectorSpark.computeRanges(hist, 41, 13);
         long[] expectedBinWidths = {1L, 1L, 1L, 7L, 7L, 7L, 9L, 11L, 11L, 11L};
         Assert.assertEquals(calculatedBinWidths, expectedBinWidths);
     }
```

Comment:
Could you add small unit tests for the levelUp() and convertSortedMapToHTSHistogram() methods?